### PR TITLE
Testing db

### PR DESCRIPTION
**Pull request by ZilvinasVysniauskas**

### CODE REVIEW

File: src/main/java/com/example/desk_reservation_app/controller/AdminPageController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    private final DeskReservationService deskReservationService;
+    private final UserService userService;
+
+    public AdminPageController(DeskReservationService deskReservationService, UserService userService) {
+        this.deskReservationService = deskReservationService;
+        this.userService = userService;
+    }
+
+    @PostMapping("/user")
+    public void AddOrEditUser(@RequestBody UserRequest userRequest) {
```
Comment (MeRytis) [inline src/main/java/com/example/desk_reservation_app/controller/AdminPageController.java:25]: Usually when word _"and"_ or _"or"_ appears in method name that is a sign that it should be split into 2 different methods. I would suggest to do that in this case.

File: src/main/java/com/example/desk_reservation_app/dto/mappers/desk/ReservationsMapper.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import com.example.desk_reservation_app.dto.api.desks.RoomDto;
+import com.example.desk_reservation_app.dto.requests.ReservationRequest;
+import com.example.desk_reservation_app.models.Desk;
+import com.example.desk_reservation_app.models.Reservations;
+import com.example.desk_reservation_app.models.Room;
+import com.example.desk_reservation_app.models.User;
+
+import java.util.stream.Collectors;
+
+public class ReservationsMapper {
```
Comment (MeRytis) [inline src/main/java/com/example/desk_reservation_app/dto/mappers/desk/ReservationsMapper.java:15]: Please check each method name in this class. According to naming convention:
_Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized._

File: src/main/java/com/example/desk_reservation_app/dto/mappers/user/UserMapper.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.example.desk_reservation_app.dto.mappers.user;
+
+import com.example.desk_reservation_app.dto.api.user.UserAdminDto;
+import com.example.desk_reservation_app.dto.requests.UserRequest;
+import com.example.desk_reservation_app.models.User;
+
+public class UserMapper {
```
Comment (MeRytis) [inline src/main/java/com/example/desk_reservation_app/dto/mappers/user/UserMapper.java:7]: Please check each method name in this class. According to naming convention:
_Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized._

File: src/main/java/com/example/desk_reservation_app/services/DeskReservationService.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    private final UserRepository userRepository;
+
+    public DeskReservationService(FloorRepository floorRepository, ReservationsRepository reservationsRepository, DeskRepository deskRepository, UserRepository userRepository) {
+        this.floorRepository = floorRepository;
+        this.reservationsRepository = reservationsRepository;
+        this.deskRepository = deskRepository;
+        this.userRepository = userRepository;
+    }
+
+    public List<RoomDto> getTablesByDate(LocalDate date) {
```
Comment (MeRytis) [inline src/main/java/com/example/desk_reservation_app/services/DeskReservationService.java:37]: In some cases you use word _Desk_ while in other case word _Table_ is used. I would suggest to unify this in order to do not confuse.

**Review by MeRytis — APPROVED** [review]
Are you going to add Swagger API documentation here? If yes, then would be good to add description of each API service as well. 
What about Unit tests? Will you cover your code with those? 
The rest looks very well, good work!

File: src/main/java/com/example/desk_reservation_app/controller/AdminPageController.java (hunk 1 above)
Comment (ZilvinasVysniauskas) [reply src/main/java/com/example/desk_reservation_app/controller/AdminPageController.java:25]: On it already, we figured that having one method could cause problems when admin would add new user but enter already existent id, so two different methods are required for this.

> Are you going to add Swagger API documentation here? If yes, then would be good to add description of each API service as well. What about Unit tests? Will you cover your code with those? The rest looks very well, good work!

Thank you for your feedback, we still don't have api documentations, but now all api are clear so I will right it and add it. I would be nice to have unit test, though the lack of time strongly discourages  me to focus anywhere else apart main functionality. Though avoiding writing test could bring us bigger issues at the end of month so we could have additional task for this sprint to cover this code with unit tests

File: src/main/java/com/example/desk_reservation_app/services/DeskReservationService.java (hunk 4 above)
Comment (ZilvinasVysniauskas) [reply src/main/java/com/example/desk_reservation_app/services/DeskReservationService.java:37]: All naming's in this project needs to be reconsidered. I was rushing to create api for front end, so we would have to mock data, so there are some names which still require some more thoughts naming then. I am not particularly keen on naming of the mappers too.

**Pull request merged**